### PR TITLE
CF Transactor(initial)

### PR DESCRIPTION
**Pull request by haiodo**
CF Transactor(initial)

### CODE REVIEW

File: workers/transactor/wrangler.toml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+# Docs: https://developers.cloudflare.com/workers/configuration/smart-placement/#smart-placement
+# [placement]
+# mode = "smart"
+
+# Variable bindings. These are arbitrary, plaintext strings (similar to environment variables)
+# Docs:
+# - https://developers.cloudflare.com/workers/wrangler/configuration/#environment-variables
+# Note: Use secrets to store sensitive data.
+# - https://developers.cloudflare.com/workers/configuration/secrets/
+[vars]
```
Comment (aonnikov) [inline workers/transactor/wrangler.toml:23]: please add dev/staging configs, or just staging with dev as default

File: workers/transactor/wrangler.toml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+ACCOUNTS_URL = "http://127.0.0.1:3000"
+SERVER_SECRET = "secret"
```
Comment (aonnikov) [inline workers/transactor/wrangler.toml]: this should be a secret, not variable

File: workers/transactor/src/transactor.ts [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+
+    registerServerPlugins()
+
+    this.accountsUrl = env.ACCOUNTS_URL
+
+    this.measureCtx = new MeasureMetricsContext('transactor-' + this.ctx.id.toString(), {})
+
+    setMetadata(serverPlugin.metadata.Secret, env.SERVER_SECRET)
+
+    console.log('Connecting DB to', env.HYPERDRIVE.connectionString)
```
Comment (aonnikov) [inline workers/transactor/src/transactor.ts]: nit: cloudflare recommends use objects for logging:

```
console.log({ message: 'Connecting DB to', hyperdrive: env.HYPERDRIVE.connectionString })
```

Comment (haiodo) [reply workers/transactor/src/transactor.ts]: Will do

**Review by aonnikov — APPROVED** [review]

**Pull request merged**